com.unity.multiplayer.mlapi/Runtime/Core/NetworkBehaviour.cs

```diff
@@ -518,7 +518,7 @@ internal static void NetworkBehaviourUpdate(NetworkManager networkManager, Snaps
                             // Sync just the variables for just the objects this client sees
                             for (int k = 0; k < sobj.ChildNetworkBehaviours.Count; k++)
                             {
-                                sobj.ChildNetworkBehaviours[k].VariableUpdate(client.ClientId, snapshot);
+                                sobj.ChildNetworkBehaviours[k].VariableUpdate(client.ClientId, networkManager.SnapshotSystem);
```

Comment:
NetworkBehaviour.cs already has a `NetworkManager` member variable, so I don't think you have to pass this in.